src/platform/Linux/CHIPBluezHelper.cpp

```diff
@@ -1295,8 +1295,9 @@ static void UpdateAdditionalDataCharacteristic(BluezGattCharacteristic1 * charac
 
     writer.Finalize();
```

Comment:
Not new, but this should check for errors, right?  File a followup, please?

Reply:
Done now given the revised Finalize().